text/0003-book-participation.md

```diff
@@ -0,0 +1,93 @@
+# Participation in writing the book
+
+This RFC aims to lay out how participating in authoring the book works.
+
+## Details
+
+I would like for the book to not be solely written by myself. That said, when
+I started this project, I wasn’t sure if anyone would even want to help out,
+and I had a plan. Including others means that plan may change.
+
+Specifically, here is my proposal:
+
+* There are two groups of contributors to the book: editors and authors.
+* All editors are authors, but not all authors are editors.
+* Editors have a commit bit, authors must send in PRs.
+* Initially, Steve will be the sole editor, but if others are interested, we’ll
```

Comment:
We should probably change this to reflect the two person editorial team mentioned below.